packages/tools/migrations-irec/config/seed.sql

```diff
@@ -664,4 +664,75 @@ VALUES (
         'd49f7fed-2f79-4d10-8985-41a0c3e9ba03',
         'METER003',
         'Biomass Facility description'
-    );
\ No newline at end of file
+    );
+
+INSERT INTO public."platform_organization" (
```

Comment:
it's minor but maybe https://www.postgresqltutorial.com/postgresql-insert-multiple-rows/
would be more readable?

Reply:
fixed